### PR TITLE
SC22 added CUDA Graphs hands-on and housekeeping.

### PR DESCRIPTION
**Pull request by jirikraus**

### CODE REVIEW

**Comment by jirikraus** [comment]
@AndiH this is good to go from my perspective so feel free to merge if you don't see an issue.

Copied @mhrywniak for visibility.

File: 10B-H_Using_CUDA_Graphs/.master/Instructions.md [hunk 1]
```diff
@@ -0,0 +1,40 @@
...
+    https://sc22.supercomputing.org/presentation/?id=tut102&sess=sess196
+
+## Hands-On 10B: Using CUDA Graphs
+
+### Task: Combining CUDA Graphs with NCCL for Inter-GPU Communication
+
+#### Description
+
+The purpose of this task is to introduce [CUDA Graphs](https://docs.nvidia.com/cuda/cuda-c-programming-guide/index.html#cuda-graphs).
+For that the NCCL version of the Jacobi solver developed in hands-on 8 is modified to to use the
```
Comment (AndiH) [inline 10B-H_Using_CUDA_Graphs/.master/Instructions.md:15]: `For that, the`

`to to`

**Comment by AndiH** [comment]
Apart from the two typos, this looks good to me. Did not test, though.

File: 10B-H_Using_CUDA_Graphs/.master/Instructions.md (hunk 1 above)
Comment (jirikraus) [reply 10B-H_Using_CUDA_Graphs/.master/Instructions.md]: Thanks fixed them.

**Pull request merged**